composer.json

```diff
@@ -16,13 +16,13 @@
         }
     ],
     "require": {
-        "php": "^7.1 || ^7.3 || ^8.0",
-        "aws/aws-sdk-php": "^3.145",
-        "illuminate/support": "5.7.* || 5.8.* ||^6.0 || ^7.0 || ^8.0"
+        "php": "^7.4 || ^8.0",
+        "aws/aws-sdk-php": "^3.219",
+        "illuminate/support": "^6.0 || ^7.0 || ^8.0 || ^9.8"
     },
     "require-dev": {
-        "orchestra/testbench": "^3.8 || ^5.0 || ^6.0",
-        "phpunit/phpunit": "^7.0 || ^8.4 || ^9.3.3"
+        "orchestra/testbench": "^7.4",
```

Comment:
I think you need multiple versions here. I approved the actions to run and you can see which ones are failing now.